libbpf-rs/CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ Unreleased
 - Added `OpenProgramMut::set_autoattach`
 - Adjusted `UprobeOpts::func_name` to be an `Option`
 - Implemented `Sync` for `Link`
+- Added kprobe multi support for attaching programs, with and without providing
+  additional options.
+- Allow to provide additional options when attaching programs to raw
+  tracepoints.
+- Allow to provide additional options when attaching programs to kprobes.
```

Comment:
Nit: Let's order these to the top please. I try to order items by the some subjective "impact"/size of change and these are most certainly bigger additions than the few member additions above. Please also remove trailing periods `.` to stick with existing style.